### PR TITLE
Improved `ROT13` function

### PR DESCRIPTION
**Pull request by fahimfaisaal**
# Welcome to the JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

## Merged my master branch

- [ ] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?
- [x] Added test case

### Checklist:

- [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [ ] This pull request is all my own work -- I have not plagiarized.
- [ ] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new JavaScript files are placed inside an existing directory.
- [ ] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,4 +35,4 @@
     "jest": "^26.4.2",
     "standard": "^16.0.4"
   }
-}
+}
```
Comment (raklaptudirm) [inline package.json:38]: Why did you remove the newline?

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (fahimfaisaal) [reply package.json:38]: my editor by default removes that line on save, wait just a second.

**Review by raklaptudirm — APPROVED** [review]
